### PR TITLE
Updated contributing guidelines

### LINKED ISSUE #14: Update README
Your README currently looks great! I would recommend adding:

* How users can setup the project on their local machine for contributions.
* Explaining to users you have a LEARN.md file
* Mention your CONTRIBUTING file
* Mention your code of conduct

### PR DESCRIPTION
**Pull request by PragatiVerma18**
Fixes #14 
---
- Updated README to mention Contributing Guidelines and Code of Conduct
- Updated Contributing Guidelines

### CODE REVIEW

**Comment by PragatiVerma18** [comment]
@ericdrosado please check and let me know if this works.

**Comment by ericdrosado** [comment]
Could we also add a fix for this broken image in this PR:

Screen Shot 2022-05-25 at 2 22 31 PM

**Comment by PragatiVerma18** [comment]
Done @ericdrosado, please check once and approve.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,19 +1,114 @@
...
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
 
+### Setting things up
+
+To set up the development environment, follow the instructions in README.
+
+### Finding something to work on
+
+The issue tracker of EvalAI a good place to start. If you find something that interests you, comment on the thread and we’ll help get you started.
```
Comment (ericdrosado) [inline CONTRIBUTING.md]: I see a mention to `EvalAI` here 😬

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,19 +1,114 @@
...
+
+Alternatively, if you come across a new bug on the site, please file a new issue and comment if you would like to be assigned. The existing issues are tagged with one or more labels, based on the part of the website it touches, its importance etc., that can help you in selecting one.
+
+If neither of these seem appealing, please post on our channel and we will help find you something else to work on.
+
+### Instructions to submit code
+
+Before you submit code, please talk to us via the issue tracker so we know you are working on it.
+
+Our central development branch is development. Coding is done on feature branches based off of development and merged into it once stable and reviewed. To submit code, follow these steps:
```
Comment (ericdrosado) [inline CONTRIBUTING.md]: I noticed your primary branch is not `development`.

**Comment by ericdrosado** [comment]
@PragatiVerma18 it looks like your `CONTRIBUTING.md` is from another repo. That's fine, but please review it to ensure that all the information included is relevant to your project.

File: CONTRIBUTING.md (hunk 2 above)
Comment (PragatiVerma18) [reply CONTRIBUTING.md]: Changed this as well. Please check now.

**Comment by PragatiVerma18** [comment]
> @PragatiVerma18 it looks like your `CONTRIBUTING.md` is from another repo. That's fine, but please review it to ensure that all the information included is relevant to your project.

Hey @ericdrosado, I have updated the `CONTRIBUTING.md` as per StandNote, please check once again and give us your feedback.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,19 +1,104 @@
...
 
 When contributing to this repository, please first discuss the change you wish to make via issue,
-email, or any other method with the owners of this repository before making a change. 
+email, or any other method with the owners of this repository before making a change.
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
 
+### Setting things up
+
+To set up the development environment, follow the instructions in README.
```
Comment (ericdrosado) [inline CONTRIBUTING.md]: it would be a great user experience to add a hyperlink to README.

**Review by ericdrosado — APPROVED** [review]
Other than the one comment I left, these changes looks good! Feel free to merge once you've addressed the comment. 🚢

File: CONTRIBUTING.md (hunk 3 above)
Comment (PragatiVerma18) [reply CONTRIBUTING.md]: Done

**Pull request merged**